test/controllers/user.test.js

```diff
@@ -0,0 +1,86 @@
+const app = require('../../app'),
+  request = require('supertest');
+
+const sampleUser = { name: 'nicolas', lastName: 'L', email: 'freecandy@msn.net', password: 'abc452626' };
+const sameEmailUser = {
+  name: 'F',
+  lastName: 'Perez',
+  email: 'freecandy@msn.net',
+  password: 'abc351515'
+};
+const invalidPasswordUser = {
+  name: 'M',
+  lastName: 'M',
+  email: 'mm@cgt.com',
+  password: '123'
+};
+const mentallyChallengedUser = {
+  email: 'yes'
+};
+
+describe('users api', () => {
```

Comment:
You could use something like `/users POST` because all endpoints of users are part of user api